src/Servers/Kestrel/Core/src/Internal/Http/Http1ChunkedEncodingMessageBody.cs

```diff
@@ -109,6 +109,13 @@ public override void Complete(Exception exception)
             _context.ReportApplicationError(exception);
         }
 
+        public override ValueTask CompleteAsync(Exception exception)
+        {
+            _completed = true;
+            _context.ReportApplicationError(exception);
+            return new ValueTask();
+        }
```

Comment:
I don't think this needs to be overridden. Doesn't the default version of CompleteAsync now do exactly this?